en/developer_guide/esi.rst

```diff
@@ -85,6 +85,17 @@ The generic tags are added for all kind of blocks, they depend on :
 Additionally, each rendering strategy for blocks can define a list of specific tags
 by implementing the method ``DisplayBlockInterface::getCacheTags()``.
 
+By default, Open Orchestra provides some tag that can be used in the rendering strategies:
+
+* ``node- + NODE_ID`` : Invalidate when node status is updated.
+* ``contentType- + CONTENT_TYPE`` : Invalidate when content type is updated or deleted.
+* ``contentId- + CONTENT_ID`` : Invalidate when content status is updated or content is deleted.
+* ``mediaId- + MEDIA_ID`` : Invalidate when media is updated or deleted.
+* ``menu- + SITE_ID`` : Invalidate when a node of this site is deleted, restored, his status changed or his path updated.
+
+These tags are gathered in the ``OpenOrchestra\BaseBundle\Manager\TagManager`` class.
+
+If this tags isn't enough,
```

Comment:
It's possible to define new cache tags which would be flushed by calling
 ``OpenOrchestra\DisplayBundle\Manager\CacheableManager::invalidateTags()``